cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -1084,6 +1205,10 @@ void ICACHE_FLASH_ATTR *umm_info( void *ptr, int force ) {
       ummHeapInfo.maxFreeContiguousBlocks = curBlocks;
     }
   }
+#if defined(DEBUG_ESP_PORT)
```

Comment:
This is specific to the ESP, and won't be accepted as-is upstream.I'm not sure what would be the best approach, maybe discussing asserts with the original author should be done first.

Reply:
The asserts were for initial verification of the new method for tracking freeheap. It compared the new with the original method. I think it can now be removed. As well as the include <assert.h>